### PR TITLE
Pyfiglet GUI

### PR DESCRIPTION
**Pull request by bhoran**
A Small demo of accessing a Python class from Java.
1. Installs the pyfiglet module
2. Creates a Python class that acts as a wrapper for the pyfiglet module
3. Creates a Java interface that acts as a proxy for the wrapper
4. Creates a Swing JFrame that calls the proxy's _format_ method

From discussion with @timfel , this may be the wrong approach, and I'm not at all sure about the _pom.xml_ file.
All suggestions gratefully received (and please ignore the README for the time being)
Pyfiglet GUI

### CODE REVIEW

File: graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    // data that
+    // is explicitly shared and instead use methods and their return values, similar
+    // to how one
+    // would limit the visibility of classes within a Java project.
+    Value pyfigletWrapperClass = context.getPolyglotBindings().getMember("PyfigletWrapper");
+    // Next we instantiate the Python type and cast it to a GraphRenderer. This cast
+    // will
+    // always succeed, and the relevant methods will only be forwarded when invoked,
+    // so there
+    // is typechecking at this point, even at runtime. The reason is that Python
```
Comment (timfel) [inline graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java:122]:
```suggestion
    // is no typechecking at this point, even at runtime. The reason is that Python
```

File: graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        option("python.ForceImportSite", "true").build();
+    InputStreamReader code = new InputStreamReader(Main.class.getClassLoader().getResourceAsStream(SOURCE_FILE_NAME));
+    Source source;
+    try {
+      source = Source.newBuilder(PYTHON, code, SOURCE_FILE_NAME).build();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    context.eval(source);
+    // Getting the Python PyfigletWrapper class is an example of how data can be
```
Comment (timfel) [inline graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java:110]: The linebreaks are formatted strangely

**Review by timfel — CHANGES_REQUESTED** [review]
This is a nice demo :)

File: graalpy-install-package/README.md [hunk 3]
```diff
@@ -1,18 +1,36 @@
...
-For more details on polyglot embedding please see the docs:
-https://www.graalvm.org/latest/reference-manual/embed-languages/
+This small application demonstrates how to use GraalPy to access a Python class from a Java application, built with Maven.
 
-## Setup
+It includes the following steps:
+1. Install the [pyfiglet](https://github.com/pwaller/pyfiglet) module. (Pyfiglet is an ASCII art module to create styled text.)
+2. Create a Python class ([PyfigletWrapper.py](src/main/resources/PyfigletWrapper.py)) that acts as a wrapper for the pyfiglet module.
+3. Create a Java interface ([PyfigletProxy.java](src/main/java/com/oracle/example/graalpy/PyfigletProxy.java)) that acts as a proxy for the wrapper.
+4. Create a Swing JFrame ([PyfigletFrame.java](src/main/java/com/oracle/example/graalpy/PyfigletFrame.java)) that calls the proxy's _format_ method.
```
Comment (olyagpl) [inline graalpy-install-package/README.md:9]: the proxy's `format` method.

File: graalpy-install-package/README.md [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
+# Using GraalPy to Access a Python Class from a Java Application 
 
-For more details on polyglot embedding please see the docs:
-https://www.graalvm.org/latest/reference-manual/embed-languages/
+This small application demonstrates how to use GraalPy to access a Python class from a Java application, built with Maven.
 
-## Setup
+It includes the following steps:
+1. Install the [pyfiglet](https://github.com/pwaller/pyfiglet) module. (Pyfiglet is an ASCII art module to create styled text.)
+2. Create a Python class ([PyfigletWrapper.py](src/main/resources/PyfigletWrapper.py)) that acts as a wrapper for the pyfiglet module.
```
Comment (olyagpl) [inline graalpy-install-package/README.md:7]: the pyfiglet module >  the Pyfiglet module.

File: graalpy-install-package/README.md [hunk 5]
```diff
@@ -1,18 +1,36 @@
...
+2. Run the application:
+    ```bash
+    mvn exec:exec -Prun
+    ```  
 
-Please see the [pom.xml](./pom.xml) file for further details on the configuration.
\ No newline at end of file
+
+![Pyfiglet Java Application](Pyfiglet%20GUI.gif)
+
```
Comment (olyagpl) [inline graalpy-install-package/README.md:32]: Double space

**Review by olyagpl — APPROVED** [review]

**Pull request merged**